src/main/scala/com/kinja/config/BootupErrors.scala

```diff
@@ -5,7 +5,7 @@ package com.kinja.config
  * It is also a monad (but does not accumulate errors in that case).
  */
 final case class BootupErrors[A] private[BootupErrors] (run : Either[Seq[ConfigError], A]) extends AnyVal {
-  def map[B](f : A => B) : BootupErrors[B] = BootupErrors(this.run.right.map(f))
+  def map[B](f : A => B) : BootupErrors[B] = BootupErrors(this.run.map(f))
```

Comment:
We could probably make this work using `match` in 2.11 (`Either` doesn't have `map` in 2.11 and `right` is deprecated in 2.13).